### PR TITLE
Adding Flask-CORS

### PR DESCRIPTION
**Pull request by markcatley**
Just updating the existing PR to the latest version of master.

### CODE REVIEW

**Comment by josegonzalez** [comment]
Can you add docs for this to the readme? Also, what exactly does this enable (my memory of cors is hazy at best).

**Comment by llonchj** [comment]
+1

**Comment by josegonzalez** [comment]
@markcatley still needs docs.

**Comment by markcatley** [comment]
@josegonzalez I have added a line in the readme.

File: README.rst [hunk 1]
```diff
@@ -72,6 +72,8 @@ Using the API
 
 All interaction with Sixpack is done via ``HTTP GET`` requests. Sixpack allows for cross-language testing by accepting a unique ``client_id`` (which the client is responsible for generating) that links a participation to a conversion. All requests to Sixpack require a ``client_id``.
 
+The SixPack API can be used from front-end Javascript via CORS-enabled requests. The Sixpack API server will accept CORS requests from any domain.
```
Comment (zackkitzmiller) [inline README.rst]: Minor, but Sixpack.

**Comment by markcatley** [comment]
@zackkitzmiller fixed.

**Comment by josegonzalez** [comment]
Released as 2.6.0, can you test @kuroisuna?

**Comment by kuroisuna** [comment]
It is working perfectly, thanks a lot @markcatley, @josegonzalez

**Pull request merged**